src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Distinct.txt

```diff
@@ -0,0 +1,92 @@
+// Blank table
+>> Distinct(If(1<0,[1,2]), true)
+Blank()
+
+// Empty table
+>> Distinct(Filter([1,2,3],Value>10), true)
+Table()
+
+// Constant expressions
```

Comment:
Sounds like For In-memory collections, existing JS implementation is indeed ordered.  We should assume our 1+ million customers rely on that behavior even if it’s not in the docs. 
But delegated calls are not necessarily ordered

So I'd comment about ordering and add these test cases: 

Distinct([1,2,3,2,1], Value)
1,2,3

Distinct([3,2,1,3,3,3,3], Value)
3,2,1

Reply:
added the test as mentioned.